tezos-sig-gating-contracts/scripts/tests/nftminter.spec.ts

```diff
@@ -120,52 +122,46 @@ describe(`ExampleGatedNFTMinter`, function () {
     );
 
     // MINT OFFCHAIN
-    const signerBob = new InMemorySigner(
-      "edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt"
-    ); // bob private key
-    const functioncall_contract = exampleGatedNFTMinter
+    const functionCallContractAddress = exampleGatedNFTMinter
       ? exampleGatedNFTMinter
       : "";
-    const functioncall_name = "%mint_gated";
-    const functioncall_params = {
+    const functionCallName = "%mint_gated";
+    const functionCallArgs = {
       owner: "tz1fon1Hp3eRff17X82Y3Hc2xyokz33MavFF",
       token_id: "1",
     };
-    const dataKey = "edpkurPsQ8eUApnLUJ9ZPDvu98E8VNj4KtJa1aZr16Cr5ow5VHKnz4"; // bob public key
+    const dataKey = nexeraSignerPublicKey; // signer public key
     const expiration = (currentBlock + 10).toString();
     const nonce = "0";
     const userAddress = "tz1fon1Hp3eRff17X82Y3Hc2xyokz33MavFF";
-    const chain_id = currentChainId;
+    const chainID = currentChainId;
 
     // Prepare Hash of payload
-    const functioncall_params_bytes = convert_mint(
-      functioncall_params.owner,
-      functioncall_params.token_id
+    const functionCallArgsBytes = convert_mint(
+      functionCallArgs.owner,
+      functionCallArgs.token_id
     );
     const payload_hash = compute_payload_hash_for_mint(
-      chain_id,
+      chainID,
```

Comment:
The idea was to group all these arguments into an object typed TezosTxAuthData, like in the branch I made. That was we see which arguments are needed. 

Also please rename these functions to camelCase.

Also ideally please une a props object input for this function that way it is easier and more readable to add and remove parameters